x-pack/plugins/cases/server/client/metrics/lifespan.ts

```diff
@@ -5,24 +5,165 @@
  * 2.0.
  */
 
-import { CaseMetricsResponse } from '../../../common/api';
+import { SavedObject } from 'kibana/server';
+import moment from 'moment';
```

Comment:
I think we [should not use momentjs](https://github.com/you-dont-need/You-Dont-Need-Momentjs) for simple date calculations.